src/components/color-panel/index.ts

```diff
@@ -0,0 +1 @@
+export { ColorPanel } from './color-panel';
```

Comment:
if color panel will be only in header component, then you can put this component in the header folder